src/App.js

```diff
@@ -42,11 +52,28 @@ import {
   MARKER_TEXT_PADDING,
   MARKER_HEIGHT,
   MARKER_TICK_HEIGHT,
+  REACT_PRIORITIES,
+  ROW_CSS_PIXELS_HEIGHT,
+  TEXT_CSS_PIXELS_OFFSET_START,
```

Comment:
A lot of these constants aren't being used. It may be a good idea to not import these. I think we can leave them in constants.js and clean them up in the future.